randovania/games/prime1/logic_database/Tallon Overworld.txt

```diff
@@ -609,7 +609,7 @@ Extra - aabb: [-703.2118, 297.52924, -49.04736, -619.3062, 378.77765, 64.30938]
   > Out of Bounds (Skywalk)
       All of the following:
           # https://youtu.be/imhkxmwiU9o
-          Boost Ball and Morph Ball and Power Bomb and Movement (Expert) and Single Room Out of Bounds (Advanced) and Disabled Dock Rando and Disabled Room Rando
```

Comment:
I feel like there's more instances of the resource being used? ex: phen shorelines I'm pretty sure has one for the lighthouse OoB

Reply:
I thought so too, but nope, this is the only use of this resource.
Lighthouse is `Boost Ball and Morph Ball and Scan Visor and Space Jump Boots and Combat/Scan Dash (Intermediate) and Movement (Advanced) and Single Room Out of Bounds (Advanced) and Standable Terrain (Advanced) and Disabled Dock Rando and Shoot Super Missile`

IRE is 
```
All of the following:
  Morph Ball and Clip Through Objects (Expert)
  Any of the following:
    # https://youtu.be/raoSUAIfIUk
    Space Jump Boots and L-Jump (Advanced) and Single Room Out of Bounds (Advanced) and Standable Terrain (Expert)
    All of the following:
      Morph Ball Bomb and Bomb Jump (Hypermode) and Complex Bomb Jump (Expert) and Combat/Scan Dash (Expert) and Single Room Out of Bounds (Expert)
      Scan Visor or After Research Core Power Outage or Shoot Any Beam
```

The other probably should have that resource, and I remember toaster mentioning it in the Oob PR. But adding that is out of scope for this PR.